Tasks/ipa-resign/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Azure Pipelines task to resign an ipa file",
   "main": "ipa-resign.js",
   "dependencies": {
+    "azure-pipelines-task-lib": "^2.11.1",
     "vsts-task-lib": "^2.0.0-preview"
```

Comment:
Could you please remove vsts-task-lib since it's not in use?

Reply:
Updated task-lib in tests for tasks, removed this dependency, thanks!